q2_cutadapt/tests/test_trim.py

```diff
@@ -10,7 +10,6 @@
 import itertools
 import os
 import unittest
-
```

Comment:
**Lint:**
Please add this newline back as well (separates stdlib from 3rd party imports)